proposed/simplecache-meta.md

```diff
@@ -0,0 +1,82 @@
+PSR-Simple-Cache Meta Document
+==============================
+
+1. Summary
+----------
+
+Caching is a common way to improve the performance of any project, making
+caching libraries one of the most common features of many frameworks and
+libraries. Interoperability at this level means libraries can drop their
+own caching implementations and easily rely on the one given to them by the
+framework, or another dedicated cache library.
+
+
+2. Why Bother?
+--------------
+
+PSR-6 solves this problem already, but in a rather formal and verbose way for
+what the most simple use cases need. This simpler approach aims to build a
+standardized layer of simplicity on top of the existing PSR-6 interfaces.
+
+
+3. Scope
+--------
+
+### 3.1 Goals
+
+* A very simple interface for basic cache operations.
+* A few more advanced ones for more specialized operations.
```

Comment:
A few more advanced whats?

Also, could you either adapt this to British English `specialised` or use American English throughout the entire specification (We don't require one or the other, only consistency).